recipes/rnablueprint/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: rnablueprint
+  version: "1.2.0"
+
+source:
+  fn: RNAblueprint-1.2.tar.gz
+  url: https://github.com/ViennaRNA/RNAblueprint/releases/download/1.2/RNAblueprint-1.2.tar.gz
+  md5: 3d02264f8d0dd1d15d003f4b25aafeeb
+  #git_rev: 1.2
+  #git_url: https://github.com/ViennaRNA/RNAblueprint.git
+
+build:
+  skip: True  # [not py27 or osx]
+  number: 0
+
+requirements:
+  build:
+    #-  libtool
+    -  boost >=1.55.0
+    -  swig >=3.0.7
+    -  python 2.*
+
+  run:
+    -  boost >=1.55.0
```

Comment:
you can add boost like this:
 - boost {{CONDA_BOOST}}*

Reply:
maybe you need change the LD_LIBRARY_PATH like this:
export LD_LIBRARY_PATH="$PREFIX/lib"